docs/pages/eas-update/deployment.mdx

```diff
@@ -111,13 +111,17 @@ A common practice for creating staging builds is to always create one whenever y
 
 ## Deploying to production
 
-When creating a new production build, increment your app version every time to ensure it has a unique runtime version.
+Run `eas update --channel production` to bundle and push a new update to production. This will make your hotfix immediately available to users of production builds with the same runtime version.
```

Comment:
What do you think about making this guide only mention `eas update:republish --destination-channel` instead of having both methods?

With both methods of promotion here, the read is a bit confusing since it's not clear which conditions apply to each method. For example, the following doesn't apply to the republish method:
> **If you have already published the fix to staging and verified it there**, ensure that you are republishing from the same commit.